tests/testsuite/git_auth.rs

```diff
@@ -366,7 +435,29 @@ Caused by:
 
     p.cargo("check -v")
         .with_status(101)
-        .with_stderr_contains("[..]Unable to update[..]")
+        .with_stderr_data(str![[r#"
+[UPDATING] git repository `ssh://needs-proxy.invalid/git`
+[RUNNING] `git fetch --verbose --force --update-head-ok [..]ssh://needs-proxy.invalid/git[..] [..]+HEAD:refs/remotes/origin/HEAD[..]`
+ssh: Could not resolve hostname needs-proxy.invalid: [..] not known
+fatal: Could not read from remote repository.
+
+Please make sure you have the correct access rights
+and the repository exists.
```

Comment:
I was thinking we shouldn't over matching against Git and SSH output, but meh we've already done that so leave it.